Rewrite Azure SAML docs for application owners (#2112)
clerk/clerk-docs: docs/authentication/enterprise-connections/saml/azure.mdx

    }
  ]}
>

>   - Create a SAML SSO connection for your customer

Reviewer: To prepare for the self-serve world - app owner reading this doc to customer SSO into Clerk dashboard, what do you think of removing "your customer" here

```suggestion
  - Create a SSO connection via Clerk Dashboard 
```

>   - Give your customer instructions on how to connect Microsoft Azure Entra ID to your app

Reviewer: Up to discussion: Why not using the "organization" term instead of "customer" here to align with our product modelling

This process requires configuration changes in both the Clerk dashboard, and in your customer's Microsoft Entra ID settings in their Azure account.

<Steps>
  ## Create a Microsoft Entra ID SAML connection in Clerk

  1. In the Clerk Dashboard, navigate to the [**SSO connections**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) page.
  1. Select **Add connection** and select **For specific domains or organizations**.
  1. Under **SAML**, select **Microsoft Entra ID (Formerly AD)**.
  1. Enter the **Domain**. This is the email domain of the users you want to allow to sign in to your application. Optionally, select an **Organization**.
  1. Enter the **Name**. This will be displayed on the sign-in form.
  1. Select **Add connection**. You'll be redirected to the connection's configuration page. Note that the connection is disabled by default.
  1. In the **Service Provider Configuration** section, note the **Reply URL (Assertion Consumer Service URL)** and **Identifier (Entity ID)**. These will need to be added to your customer's Microsoft Entra ID application.

>   ## Configure the third-party Identity Provider

Reviewer: ```suggestion
  ## Configure SAML application
```

What if we remove `third-party` here, curious what was your reasoning to include it! I'd say we should mention "SAML application" instead if configuring the identity provider

Identity Provider is the "whole database of identities", and SAML application is the actual federation bucket within their system to connect to the Clerk app

  Now that the enterprise connection is configured in Clerk and the Reply URL and Identifier are known, the customer's Azure application needs to be configured. At a high level, the process is:

  - Create a new enterprise app in Microsoft Azure.
  - Assign selected users or groups to that Azure application.
  - Add the Entity ID and Reply URL from Clerk to the Azure application's SAML configuration.
  - Verify that the attribute mappings are correct

>   - Obtain and share the application's metadata URL

Reviewer: ```suggestion
  - Obtain and share the Identity Provider's application's metadata URL
```

Let's be specific that this is the IdP's application metadata URL to avoid confusing with the SP metadata URL (Clerk)

    }
  ]}
>
  - Create a SSO connection via Clerk Dashboard
  - Give your customer instructions on how to connect Microsoft Azure Entra ID to your app
  - Enable and test the SAML connection
</TutorialHero>

Enabling SAML with Microsoft Azure Entra ID (formerly [Active Directory](https://learn.microsoft.com/en-us/entra/fundamentals/new-name)) allows your users to sign up and sign in to your Clerk application with their Microsoft account.

> This process requires configuration changes in both the Clerk dashboard, and in your customer's Microsoft Entra ID settings in their Azure account.

Reviewer 2: ```suggestion
This process requires configuration changes in both the Clerk Dashboard, and in your customer's Microsoft Entra ID settings in their Azure account.
```

<Steps>
  ## Create a Microsoft Entra ID SAML connection in Clerk

  1. In the Clerk Dashboard, navigate to the [**SSO connections**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) page.
  1. Select **Add connection** and select **For specific domains or organizations**.
  1. Under **SAML**, select **Microsoft Entra ID (Formerly AD)**.
  1. Enter the **Domain**. This is the email domain of the users you want to allow to sign in to your application. Optionally, select an **Organization**.
  1. Enter the **Name**. This will be displayed on the sign-in form.
  1. Select **Add connection**. You'll be redirected to the connection's configuration page. Note that the connection is disabled by default.

>   1. In the **Service Provider Configuration** section, note the **Reply URL (Assertion Consumer Service URL)** and **Identifier (Entity ID)**. These will need to be added to your customer's Microsoft Entra ID application.

Reviewer 2: to clarify: does the IT admin need to save these values to provide to the customer later?

Reviewer: The app owner needs to save these values to provide to their customer IT admin.

>   1. In the **Service Provider Configuration** section, note the **Reply URL (Assertion Consumer Service URL)** and **Identifier (Entity ID)**. These will need to be added to your customer's Microsoft Entra ID application.

Reviewer 2: ```suggestion
  1. In the **Service Provider Configuration** section, note the **Reply URL (Assertion Consumer Service URL)** and **Identifier (Entity ID)**. These will need to be added to your customer's Microsoft Entra ID application.
```

        - **Reply URL (Assertion Consumer Service URL)**: `[YOUR_CLERK_REPLY_URL]`
    1. Select **Save** at the top of the panel. Close the panel.

    **Step 4: Verify correct configuration of attributes and claims**

    We expect your SAML responses to have the following specific attributes:

    Email address (required). This is the email address that your users will use to authenticate into your app:
      - Claim name: `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress`
      - Value: `user.mail`

    First name (optional):
      - Claim name: `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname`
      - Value: `user.givenname`

    First name (optional):
      - Claim name: `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname`
      - Value: `user.surname`

    These are the defaults, and probably won't need you to change them. However, many SAML configuration errors are due to incorrect attribute mappings, so it's worth double-checking. Here's how:

    1. Still on the **Set up Single Sign-On with SAML** page, find the **Attributes & Claims** section.
    1. Select **Edit**.

>     1  Verify that the above three attributes and values are present.

Reviewer 2: ```suggestion
    1. Verify that the above three attributes and values are present.
```

    **Step 5: Share the application's metadata URL**

    1. Still on the **Set up Single Sign-On with SAML** page, find the **SAML Certificates** section.
    1. Copy the **App Federation Metadata Url**, and reply to this email with it. This is the final piece of information we need to enable SAML.

  </Copyable>

  ## Add App Federation Metadata URL in the Dashboard

>   Now that you know the Microsoft app's **App Federation Metadata URL**, you can add it to the Clerk connection, completing the SAML connection configuration.

Reviewer 2: ```suggestion
  After following the instructions in the email, your customer should have sent you the Microsoft app's **App Federation Metadata URL**. Now, you're going to add it to the Clerk connection, completing the SAML connection configuration.
```

>   1. Navigate back to the **Clerk Dashboard**. In the **Identity Provider Configuration** section, under **App Federation Metadata Url**, paste the **App Federation Metadata URL**.

Reviewer 2: ```suggestion
  1. Navigate to the [**SSO connections**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) page in the Clerk Dashboard. 
  1. Select the SAML connection.
  1. In the **Identity Provider Configuration** section, under **App Federation Metadata Url**, paste the **App Federation Metadata URL**.
```

  1. Select **Fetch & save**.

  ## Enable the connection in Clerk

  The SAML connection is ready to enable! Once enabled, all users with email addresses ending in the domain will be redirected to the IDP at sign-in and sign-up.

  > [!WARNING]
  > If there are existing users with email domains that match the SAML connection, and there is an error in the SAML configuration in Clerk or Microsoft, those users will be **unable to sign in** when the connection is enabled. If this is a concern, we recommend coordinating with your counterpart to test the connection at an off-peak time.

  To make the connection available for users of that IDP to authenticate with:

>   1. Navigate back to the Clerk Dashboard where you should still have the connection's configuration page open. If not, navigate to the SSO connections page and select the connection.

Reviewer 2: ```suggestion
  1. Navigate back to the Clerk Dashboard where you should still have the connection's configuration page open. If not, navigate to the [**SSO connections**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) page and select the connection.
```

>   1. At the top of the page, toggle on Enable connection and select Save.

Reviewer 2: ```suggestion
  1. At the top of the page, toggle on **Enable connection** and select **Save**.
```
